### PR TITLE
Ambassador for Centraldashboard

### PR DESCRIPTION
**Pull request by swiftdiaries**
Addresses: https://github.com/kubeflow/kubeflow/issues/528
1) Adds ambassador integration for centraldashboard
2) Routes centraldashboard at "/" so that it becomes the homepage.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/622)

### CODE REVIEW

**Comment by gaocegege** [comment]
/assign @jlewi 

Sorry, I am not sure about the Centraldashboard thus I can not help you to review the PR.

File: kubeflow/core/centraldashboard.libsonnet [hunk 1]
```diff
@@ -23,52 +23,25 @@
...
-                  {
-                    name: "CENTRALDASHBOARD_NAMESPACE",
-                    valueFrom: {
-                      fieldRef: {
-                        apiVersion: "v1",
-                        fieldPath: "metadata.namespace",
-                      },
-                    },
-                  },
+                image: "swiftdiaries/centraldashboard:0.3",
```
Comment (jlewi) [inline kubeflow/core/centraldashboard.libsonnet]: Is there any issue to start publishing this image some where? If so lets link to the issue here.

Comment (swiftdiaries) [reply kubeflow/core/centraldashboard.libsonnet]: https://github.com/kubeflow/kubeflow/issues/527

Comment (swiftdiaries) [reply kubeflow/core/centraldashboard.libsonnet]: I'll add a commit with the changes.

Comment (jlewi) [reply kubeflow/core/centraldashboard.libsonnet]: Can you add a TODO and reference the GitHub issue please?

Comment (swiftdiaries) [reply kubeflow/core/centraldashboard.libsonnet]: Thank you the review.

Comment (jlewi) [reply kubeflow/core/centraldashboard.libsonnet]: As soon as you add the TODO this should be ready to go.

**Comment by jlewi** [comment]
Can you rebase please and fix the failing test? The diff chose 9k files changed and that's probably just an indication you need to rebase off master.

**Pull request closed without merging**